definitions/ext-deeper_connect/definition.yml

```diff
@@ -30,4 +30,5 @@ dashboardTemplates:
   newRelic:
     template: dashboard.json
 configuration:
-  alertable: true
\ No newline at end of file
+  alertable: true
+  entityExpirationTime: QUARTERLY
```

Comment:
how is this related to flex-host ?

Reply:
Was accidental, it can be quarterly, but yes it is unrelated to the PR in question, I will remove it on the next commit.

Reply:
Yes please. let's keep this flex related and if you need to we can do a new PR with that change

Reply:
reverted back to original